package Unit14;
public class EX01_NoCatchException
{  
    public static void main(String[] args)
    {
        int i;
       
        for( i=4; i>-2; i-- )
        {
            System.out.println("計算結果: " + 12/i);
        }
        System.out.println("程式結束!");
    }
}